spec/javascripts/NewService/ServiceSourceForm.spec.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Enzyme, {shallow} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import {ServiceSourceForm} from 'NewService'
+
+Enzyme.configure({adapter: new Adapter()})
+
+describe('Service Source Form', () => {
```

Comment:
Since you're including a describe block by default, perhaps it would help to use a more descriptive name for the specs. `ServiceSourceForm` doesn't mean much itself.